lib/repo/repository.rb

```diff
@@ -165,6 +165,11 @@ def get_permissions(user_id)
       raise NotImplementedError, "Repository.get_permissions: Not yet implemented"
     end
 
+    # Generate and write the SVN authorization file for the repo.
+    def self.__generate_authz_file
+      raise NotImplementedError, "Repository.__generate_authz_file: Not yet implemented"
```

Comment:
Line is too long. [88/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.